app/Http/Controllers/TeamController.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Team;
+use App\User;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Foundation\Auth\AuthenticatesUsers;
+use Illuminate\Http\Request;
+
+class TeamController extends Controller
+{
+	const ROLES = 'enum.user_roles';
+
+	public function __construct() {
+		$this->middleware('auth');
+	}
+
+	/**
+	 * Display a listing of the resource.
+	 *
+	 * @return \Illuminate\Http\Response
+	 */
+	public function index() {
+		//
+	}
+
+	/**
+	 * Show the form for creating a new resource.
+	 *
+	 * @return \Illuminate\Http\Response
+	 */
+	public function create() {
+		$students = User::where('role', config(self::ROLES)['student'])
```

Comment:
Igual y la vale la pena agregar este método al modelo. Siento que es algo que estamos haciendo over and over again en el codebase y que podría optimizarse y centralizarse